README.md

```diff
@@ -10,27 +10,11 @@ Internally, the Conjur Buildpack uses [Summon](https://cyberark.github.io/summon
 
 ## Requirements
 
-+ The Conjur Buildpack **requires** the `meta-buildpack` to work.
++ The Conjur Buildpack **requires** the `meta-buildpack` to work as a buildpack otherwise you will need to manually load the buildpack scripts.
 
 + Ensure that your app is bound to a Conjur service instance. For more information on binding your application to a Conjur service instance, see the [Conjur Service Broker documentation](https://github.com/conjurinc/conjur-service-broker#binding-your-application-to-the-conjur-service)**
```

Comment:
Why does this line end in a double star? I thought it was referring me somewhere else, but it's the only double star in the file.